sdk/src/main/scala/io/horizen/account/websocket/WebSocketAccountServer.scala

```diff
@@ -47,7 +47,7 @@ class WebSocketAccountServer(wsPort: Int)
       websocket.onSuccessfulTransaction(tx)
     case ChangedVault(_) =>
       websocket.onChangedVault()
-    case MempoolReAddedTransactions(readdedTxs: Seq[EthereumTransaction]) =>
+    case NewExecTransactionsEvent(readdedTxs: Seq[EthereumTransaction]) =>
       websocket.onMempoolReaddedTransaction(readdedTxs)
```

Comment:
I think, the websocket part must be renamed as well.
@MarcoOl94

Reply:
Done

Reply:
i see the same as before.
My expectation to rename `websocket.onMempoolReaddedTransaction` -> `websocket.onNewExecTransactions`
does it have sense?

Reply:
Sorry, I misunderstood the comment. Now it should be correct.